test/per_file_coverage.sh

```diff
@@ -16,7 +16,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/matcher:94.6"
 "source/common/network:94.4" # Flaky, `activateFileEvents`, `startSecureTransport` and `ioctl`, listener_socket do not always report LCOV
 "source/common/network/dns_resolver:91.4"  # A few lines of MacOS code not tested in linux scripts. Tested in MacOS scripts
-"source/common/quic:93.4"
```

Comment:
I'm surprised to see this reduced. Are there some extra unit tests that we could write to cover any new codepaths?

Reply:
There's a few error paths that I believe are unreachable, but I'm nervous to remove in case the code changes or something. Basically, some errors now get handled in the tls ContextImpl so the path in quic can't be reached.

Reply:
Makes sense.